tests/manage/mcg/ui/test_verify_region_list.py

```diff
@@ -0,0 +1,66 @@
+import re
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import tier3, polarion_id, bugzilla
+from ocs_ci.ocs.constants import NOOBAA_REGIONS_CODE_URL, AWS_REGIONS_DOC_URL
+from ocs_ci.ocs.ui.views import locate_aws_regions, locate_noobaa_regions
+from ocs_ci.ocs.ui.base_ui import BaseUI
+
+logger = logging.getLogger(__name__)
+
+
+class NavigateWebURL(BaseUI):
+    def __init__(self):
+        super().__init__()
+
+    def open_url(self, web_url):
```

Comment:
Pls have a finalizer or teardown to close the browser, better to use conftext fixtures